### PR TITLE
Add a general getting help page

### PR DESCRIPTION
**Pull request by feanil**
I could imagine using this multiple times so making it a separate page that we reference.

### CODE REVIEW

File: source/other/getting_help.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+The community also has a `slack workspace
+<https://openedx-slack-invite.herokuapp.com/>`_ where we coordinate more timely
+tasks.If you need help, we recommend posting in the forums over posting in
+slack.  The forums retain more history and tend to get more thoughtful
+responses.
+
+.. note::
+
+   The Open edX community has agreed apon a shared `code of conduct
+   <https://openedx.org/code-of-conduct/>`_, be sure to review it.
```
Comment (sarina) [inline source/other/getting_help.rst:20]:
```suggestion
   <https://openedx.org/code-of-conduct/>`_, please be sure to review it.
```

File: source/other/getting_help.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The community also has a `slack workspace
+<https://openedx-slack-invite.herokuapp.com/>`_ where we coordinate more timely
+tasks.If you need help, we recommend posting in the forums over posting in
+slack.  The forums retain more history and tend to get more thoughtful
+responses.
+
+.. note::
+
+   The Open edX community has agreed apon a shared `code of conduct
```
Comment (sarina) [inline source/other/getting_help.rst:19]:
```suggestion
   The Open edX community has agreed upon a shared `code of conduct
```

**Review by sarina — APPROVED** [review]

**Comment by markH-Docemus** [comment]
@feanil   LGTM.  1 issue - the feedback form is what I created in my google account.  at some point we should switch it to be in a TCRIL account.

**Comment by feanil** [comment]
@markH-Docemus thanks for the reminder, I created https://github.com/openedx/docs.openedx.org/issues/58 to follow up on that.

**Pull request merged**